tests/test_cli.py

```diff
@@ -42,11 +42,24 @@ def test_version_command_works(self):
         assert result.exit_code == 0
         assert atldld.__version__ in result.output
 
-    def test_cache_folder_command_works(self):
+    def test_cache_command_works(self):
         runner = CliRunner()
-        result = runner.invoke(info, ["cache-folder"])
+        result = runner.invoke(info, ["cache"])
         assert result.exit_code == 0
-        assert "global cache folder" in result.output.lower()
+        assert "atldld cache" in result.output.lower()
+
+    def test_cache_command_shows_xdg(self, monkeypatch, tmpdir):
+        runner = CliRunner()
+
+        monkeypatch.delenv("XDG_CACHE_HOME")
```

Comment:
Just to understand, what does it do exactly ? If the environment variable `XDG_CACHE_HOME` exists, is it going to be removed ?

Reply:
Yes, I think so, but just for this test.

The `XDG_CACHE_HOME` variable is set for every test automatically in the new `custom_cache_dir` fixture in `conftest.py`. So to test the case when it's actually not set one has to unset it explicitly like above.

Reply:
Thanks for your reply!